PullRequestCreated/PullRequestCreated.ts

```diff
@@ -30,7 +30,17 @@ class PullRequestCreated extends OctokitAction {
   private async newIssueParameters(pr: PullRequest): Promise<IssueParameters> {
     const mentionedIssues = this.findMentionedIssues(pr);
     console.log(`Found mentioned issues: ${mentionedIssues}`);
-    const parent = mentionedIssues.length === 1 ? await this.jira.getIssue(mentionedIssues[0]) : null;
+    let parent = mentionedIssues.length === 1 ? await this.jira.getIssue(mentionedIssues[0]) : null;
+    if (mentionedIssues.length > 1) {
+      console.log('Looking for an Epic.');
+      for (const issueKey of mentionedIssues) {
+        const issue = await this.jira.getIssue(issueKey);
+        if (issue != null && issue.fields.issuetype.name !== 'Sub-task') {
```

Comment:
As discussed [here](https://github.com/SonarSource/gh-action-lt-backlog/pull/104#discussion_r1697994366), I am taking the first non-`Sub-task`